custom/conf/app.example.ini

```
@@ -2573,3 +2573,9 @@ LEVEL = Info
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; storage type
 ;STORAGE_TYPE = local
+
+;[sync]
+;; Lock service type, could be memory or redis
+;LOCK_SERVICE_TYPE = memory
+;; Ignored by memory type, for redis, it likes "addrs=127.0.0.1:6379 db=0"
```

```suggestion
;; Ignored for the "memory" type. For "redis" use something like "addrs=127.0.0.1:6379 db=0"
```

---

Wait, that or
```suggestion
;; Ignored for the "memory" type. For "redis" use something like `redis://127.0.0.1:6379/0`
```
?
cheatsheet and example config are not in sync at the moment…